assets/react/v3/entries/course-builder/components/curriculum/Topic.tsx

```diff
@@ -383,7 +383,7 @@ const Topic = ({ topic, onDelete, onCopy, onSort, onCollapse, isOverlay = false
 										props: {
 											title: __('Lesson', 'tutor'),
 											icon: <SVGIcon name="lesson" width={24} height={24} />,
-											subtitle: __(`Topic: ${topic.post_title}`, 'tutor'),
+											subtitle: `${__('Topic:', 'tutor')}  ${topic.post_title}`,
```

Comment:
I am not sure, but I think this also does not work. Please contact with sazedul vai about this.